opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp

```diff
@@ -598,7 +603,7 @@ namespace Opm {
 
         const ADB tr_mult = transMult(state.pressure);
         const ADB mu    = fluidViscosity(canonicalPhaseIdx, phasePressure[canonicalPhaseIdx], state.temperature, state.rs, state.rv, cond);
-        sd_.rq[phase].mob = tr_mult * kr[canonicalPhaseIdx] / mu;
+        sd_.rq[phase].mob = tr_mult * rq[phase].kr / mu;
```

Comment:
ReservoirResidualQuant holds only active phases, and not all canonical phases (see line ~295 of this file)